airflow/api_fastapi/core_api/serializers/connections.py

```diff
@@ -24,9 +24,12 @@
 from airflow.utils.log.secrets_masker import redact
 
 
+# Response Models
 class ConnectionResponse(BaseModel):
     """Connection serializer for responses."""
 
+    """Connection serializer for responses."""
+
```

Comment:
Copy & Paste Error?

Reply:
Yes, exactly.  I split the models first in #43102 which has the same mistake. :facepalm: You saved me twice :pray: going to fix that PR as well.